processes_notwindows.go

```diff
@@ -62,28 +63,28 @@ func processesFromProc() ([]ProcStat, error) {
 		pid, err := strconv.Atoi(string(stats[0]))
 		if err != nil {
 			log.Errorf("Failed to convert PID(%s) to int: %s", stats[0], err.Error())
+			continue
 		}
 
-		comm, err := readProcFile(getHostProc() + "/" + string(stats[0]) + "/comm")
+		ppid, err := strconv.Atoi(string(stats[4]))
 		if err != nil {
-			log.Errorf("Failed to read comm(%s): %s", stats[0], err.Error())
+			log.Errorf("Failed to convert PPID(%s) to int: %s", stats[4], err.Error())
 		}
 
-		cmdline, err := readProcFile(getHostProc() + "/" + string(stats[0]) + "/cmdline")
-		if err != nil {
-			log.Errorf("Failed to read cmdline(%s): %s", stats[0], err.Error())
-		}
+		stat := ProcStat{PID: pid, ParentPID: ppid}
 
-		ppid, err := strconv.Atoi(string(stats[4]))
-		if err != nil {
-			log.Errorf("Failed to convert PPID(%s) to int: %s", stats[4], err.Error())
+		comm, err := readProcFile(getHostProc() + "/" + string(stats[0]) + "/comm")
```

Comment:
Maybe add a comment to explain this block a little

Reply:
It's not about the argument or converting the pid. I don't have the full context while reviewing. I just wonder what reading `.../comm` is for. I'm not sure for what `comm` is the abbreviation. Communication, command, etc.
Without a clear variable name and comment you have to know details about Linux what this is doing. Something like `Reading command information for pid` would be already sufficient.

Reply:
ignored because of inactivity